### ISSUE TITLE
Opening a single image in Sumatra automatically let the user browse the entire folder.

### ISSUE THREAD

**Issue by AirplaneRandy** [sumatrapdfreader/sumatrapdf#830]
Just bringing this up again because having to drag and drop a folder into sumatra is indeed unintuitive.
This was an old problem this other user had.
http://forums.fofou.org/sumatrapdf/topic?id=3185106

**Comment by SumatraPeter** [comment]
Yup, it was only after Krzysztof told me about the folder drag-and-drop feature [here](https://github.com/sumatrapdfreader/sumatrapdf/issues/351#issuecomment-155225771) that I came to know about it, and subsequently passed it on in the old forum thread linked to above.

He also [seems reluctant](https://github.com/sumatrapdfreader/sumatrapdf/issues/352#issuecomment-155226149) to add image-specific features for some reason.

**Comment by GitHubRulesOK** [comment]
@kjk possible duplicate of #351

**Comment by GitHubRulesOK** [comment]
Am closing as a duplicate and addressed by comments in _Add the ability to open a folder full of images_ #351
